A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_Init.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Value is rounded to 1000 and fixed to 1 decimal
+ * @param {number | string} value
+ */
+function formatWeight(value) {
+    return (+value).toFixed(1);
+}
+
+function formatTime(timestamp) {
+    var date = new Date(+timestamp * 1000);
+    return `${date.getUTCHours().toString().padStart(2, "0")}${date.getUTCMinutes().toString().padEnd(2, "0")}`;
+}
+
+class CDUAocInit {
+    static ShowPage(mcdu) {
+        mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.AOCInit;
+        mcdu.activeSystem = 'ATSU';
+
+        let fltNbr = '_______[color]amber';
+        let originIcao = '____[color]amber';
+        let destinationIcao = '____[color]amber';
+        let ete = "____[color]amber";
+        let fob = `{small}---.-{end}[color]white`;
+        let requestButton = "INIT DATA REQ*[color]cyan";
+        let gmt = "0000[color]green";
+
+        const seconds = Math.floor(SimVar.GetGlobalVarValue("ZULU TIME", "seconds"));
+        gmt = `{small}${FMCMainDisplay.secondsTohhmm(seconds)}{end}[color]green`;
+
+        function updateView() {
+            if (mcdu.page.Current === mcdu.page.AOCInit) {
+                CDUAocInit.ShowPage(mcdu);
+            }
+        }
+
+        mcdu.refreshPageCallback = () => {
+            CDUAocInit.ShowPage(mcdu);
+        };
+        SimVar.SetSimVarValue("L:FMC_UPDATE_CURRENT_PAGE", "number", 1);
+
+        if (mcdu.simbrief.sendStatus !== "READY" && mcdu.simbrief.sendStatus !== "DONE") {
+            requestButton = "INIT DATA REQ [color]cyan";
+        }
+        if (mcdu.simbrief.originIcao) {
+            originIcao = `${mcdu.simbrief.originIcao}[color]cyan`;
+        }
+        if (mcdu.simbrief.destinationIcao) {
+            destinationIcao = `${mcdu.simbrief.destinationIcao}[color]cyan`;
+        }
+        if (mcdu.simbrief.icao_airline || mcdu.simbrief.flight_number) {
+            fltNbr = `{small}${mcdu.simbrief.icao_airline}${mcdu.simbrief.flight_number}{end}[color]green`;
+        }
+        if (mcdu.simbrief.ete) {
+            ete = `${formatTime(mcdu.simbrief.ete)}[color]cyan`;
+        }
+
+        const currentFob = formatWeight(mcdu.getFOB());
+        if (currentFob) {
+            fob = `{small}${currentFob}{end}[color]green`;
+        }
+
+        const display = [
+            ["INIT/REVIEW", "1", "2", "AOC"],
+            ["{big}FMC FLT NO{end}", "{big}GMT{end}"],
```

Comment:
Are the {big} tags necessary? Seems out of place considering these lines are usually in small text.

![image](https://user-images.githubusercontent.com/28059870/102344023-4d8af380-3f69-11eb-8e2d-2c26b781f3bd.png)

Reply:
they are usually small yes, would stick with small like every other page.